rpcserver.go

```diff
@@ -3919,6 +3919,37 @@ func (r *rpcServer) fetchPendingOpenChannels() (pendingOpenChannels, error) {
 			pendingChan.BroadcastHeight()
 		fundingExpiryBlocks := int32(maxFundingHeight) - currentHeight
 
+		// remainingConfs is the number of blocks remaining until the
+		// funding transaction reaches the required confirmation height.
+		//
+		// If the funding transaction has received at least one
+		// confirmation(i.e. its block height is known), then
+		// confirmationHeight is calculated as: funding transaction
+		// block height + required number of confirmations - 1
+		// (subtracting 1 because one confirmation is already achieved).
+		// remainingConfs is computed as the difference between
+		// confirmationHeight and currentHeight, but is never less than
+		// zero.
+		//
+		// If the funding transaction hasn't been confirmed,
+		// remainingConfs is set to the total number of confirmations
+		// required.
+		var remainingConfs int32
+		openTxBlockHeight := int32(
+			pendingChan.ShortChannelID.BlockHeight,
```

Comment:
Add a comment that zeroconf channels never show up here because they are immediately marked as open from the beginning meaning that they will not show up as pending, otherwise this would cause problems here.